bug_1292234.patch

```diff
@@ -306,6 +306,111 @@ diff --git a/devtools/server/actors/storage.js b/devtools/server/actors/storage.
  StorageActors.createActor({
    typeName: "Cache",
  }, {
+diff --git a/devtools/server/tests/browser/browser.ini b/devtools/server/tests/browser/browser.ini
+--- a/devtools/server/tests/browser/browser.ini
++++ b/devtools/server/tests/browser/browser.ini
+@@ -122,6 +122,7 @@ skip-if = e10s # Bug 1183605 - devtools/
+ [browser_storage_listings.js]
+ [browser_storage_updates.js]
+ skip-if = (verify && debug && (os == 'mac' || os == 'linux'))
++[browser_storage_webext_storage_local.js]
```

Comment:
Skip-if is part of the line before. The parts in the square brackets are sections. So that skip-if applies to the line above it, not to our test.
That test is skipped if `verify`: tests are run multiple times in a row to catch intermittence etc.

If we every land this test and it starts to fail, someone who works on testing team may add a skip-if. The failures may not be related to your patch. We keep track of tests that are failing multiple times, we mark it, so we don't back out a patch because intermittent tests are failing.